product_supplierinfo_operating_unit/models/product_supplierinfo.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Open For Small Business Ltd
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# flake8: noqa
+# as we are copying from odoo core
```

Comment:
These 2 lines probably no longer relevant, also I know its stupid because the 1 function directly relates to the field declared below, but split the product part into a seperate file.